global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "8.0.100-rc.2.23502.2"
+    "version": "8.0.100"
```

Comment:
We should probably restore `"rollForward": "latestFeature"` (and maybe `"allowPrerelease": false`).

Reply:
Sounds good, will do on the next round if you don't beat me to it :-) 👍